### DISCUSSION TITLE
moji-bake on dir-selection dialog

### DISCUSSION THREAD

**Discussion by takatu-kazusige** [mifi/lossless-cut#1301] [Q&A]
Hi.
I am using lossless-cut on Ubuntu 22.04 Japanese environment, and works well except setting of working directory.

I get "moji-bake" on directory selection diagram (every character is shown as white rectangle outlined by black line).
It does not change behavior, if I change App Language into English.
I think it will be caused by a missing of some font in my environment, but I have no idea which font is to be installed.

If anybody has some ideas, please tell me.

**Comment by mifi** [comment]
Honestly I have no idea what's causing this. Maybe it will be fixed in new versions of electron once losslesscut upgrades - #714
